src/controllers/ProfileController.php

```diff
@@ -128,8 +128,9 @@ public function actionForum()
                             }
                         }
                         if ($folderExists) {
-                            if (!empty($model->avatar)) {
-                                if (!unlink($path . DIRECTORY_SEPARATOR . $model->avatar)) {
+                            $oldAvatarPath = $path . DIRECTORY_SEPARATOR . $model->avatar;
+                            if (!empty($model->avatar) && file_exists($oldAvatarPath)) {
+                                if (!unlink($oldAvatarPath)) {

```

Comment:
Wouldn't be better to merge those two conditions?
``if (!empty($model->avatar) && file_exists($oldAvatarPath) && !unlink($oldAvatarPath)) {